components/accelerator/nvidia/query/xid/xid.go

```diff
@@ -1080,6 +1236,16 @@ Bits 4 or 5: Likely HW issue with ECC/Parity --> If seen more than 2 times on th
 Bits 21 or 22: Marginal channel SI issue. Check link mechanical connecetions. If other errors accompany, follow the resolution for those.
 
 Bits 8, 9, 12, 16, 17, 24, 28: Could possibly be a HW issue: Check link mechanical connecetions and re-seat if a field resolution is required. Run diags if issue persists.
+
+
+
+"Fire-Flyer AI-HPC: A Cost-Effective Software-Hardware Co-Design for Deep Learning" https://arxiv.org/abs/2408.14158 explains:
```

Comment:
Most common error according to the paper:

<img width="496" alt="Screenshot 2024-10-07 at 11 17 12 AM" src="https://github.com/user-attachments/assets/aad5ca7a-1d76-439d-bf31-9af54c31fea4">